cisco_aci/datadog_checks/cisco_aci/helpers.py

```diff
@@ -0,0 +1,161 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import re
+
+
+def parse_capacity_tags(dn):
+    tags = []
```

Comment:
can we add docstrings and expected input/outputs for these functions?

Reply:
yes I can for most of them pretty easily